Drivers/DMM6500_DAQ6510/DMM6500_Python_Sockets_Driver/DMM6500_Sockets_Driver.py

```diff
@@ -246,12 +246,12 @@ def SetFunction_DC_Voltage(self, *args):
         #   For channel scan measurements:
         #       1. Channel string
         #       2. Input impedance
-        if (len(args) == 0):
+        if (args):
```

Comment:
would `if not args:` work here??

Reply:
I believe you are correct. I have commit the changed to applicable areas. Can you scan through my changes again for logical consistency?